src/main/resources/templates/db_scripts/schema.json

```diff
@@ -289,6 +289,48 @@
           "targetTableAlias": "fiscalYear"
         }
       ]
+    },
+    {
+      "tableName": "groupFundFY",
```

Comment:
Maybe we should follow a convention to the table name. Some tables are named in camelCase while some with `_`. Is there a reason why `groupFundFY` is in camelCase?

Reply:
The format of the name is the same as it is used for `ledgerFY`. But to be consistent with other tables I can rename current table `groupFundFY` -> `group_fund_fiscal_year`